### PR TITLE
[17.0][IMP] document_page_access_group: Users compatibility

### PR DESCRIPTION
**Pull request by victoralmau**
FWP from 16.0: https://github.com/OCA/knowledge/pull/492

Users compatibility

Please @pedrobaeza and @carolinafernandez-tecnativa can you review it?

@Tecnativa tt48786

### CODE REVIEW

File: document_page_access_group/views/document_page.xml [hunk 1]
```diff
@@ -13,7 +14,12 @@
                     string="Security"
                     groups="document_page.group_document_manager"
                 >
-                    <field name="groups_id" />
+                    <group name="groups" string="Groups" invisible="user_ids">
+                        <field name="groups_id" nolabel="1" colspan="2" />
+                    </group>
+                    <group name="users" string="Users" invisible="groups_id">
+                        <field name="user_ids" widget="many2many_tags" colspan="2" />
```
Comment (pedrobaeza) [inline document_page_access_group/views/document_page.xml]: Why `colspan="2"` for a m2m_tags?

**Review by pedrobaeza — APPROVED** [review]

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM :)

File: document_page_access_group/views/document_page.xml (hunk 1 above)
Comment (victoralmau) [reply document_page_access_group/views/document_page.xml]: You are right, without the colspan it displays a little better (taking advantage of all the space).

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge major

**Pull request merged**